### DISCUSSION TITLE
Where can I set the `process.env` properties properly in my project?

### DISCUSSION THREAD

**Discussion by wukaipeng-dev** [facebook/docusaurus#7567] [Q&A]
I have learned that docusaurus will encode small pictures to base64 resorting webpack url-loader. The picture limitation is set to 10000 bytes by default and I want to change it. Whether Docusaurus provides an eligible approach to set `process.env.WEBPACK_URL_LOADER_LIMIT`?

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/68799055/172036402-db937c58-579d-4831-93ee-4a6524f9d6d4.png">

**Comment by Josh-Cena** [comment]
If you are running on Bash (i.e. Linux/macOS), you can change your package.json script to `"build": "WEBPACK_URL_LOADER_LIMIT=0 docusaurus build"`; if you are using Windows, you need to install `cross-env`, and change it to `"build": "cross-env WEBPACK_URL_LOADER_LIMIT=0 docusaurus build"`

**Comment by wukaipeng-dev** [comment]
1. My Environment: MacOS 12.3.1 
2. My Terminal: `zsh`
3. Change the script in package: `"build": "WEBPACK_URL_LOADER_LIMIT=0 docusaurus build",`
4. Run the command `yarn build && yarn serve -p 3001`
5. Result: Nothing work, small pictures are still encoded to `base64` format in URL.

I am trying to print the `WEBPACK_URL_LOADER_LIMIT` in the terminal:

<img width="1229" alt="image" src="https://user-images.githubusercontent.com/68799055/172037810-bc09499e-f71f-4b89-9ac6-d564d821379a.png">

<img width="400" alt="image" src="https://user-images.githubusercontent.com/68799055/172037834-da4cfa73-8573-41c5-8cfc-a843b2fffb67.png">

That shows the `WEBPACK_URL_LOADER_LIMIT=0` is set successfully.

**Reply by wukaipeng-dev** [reply]
@Josh-Cena  Please have look at whether by my mistake or any other reasons.

**Comment by HesterG** [comment]
Hi, have you figured out a right way to do so? I also encountered the same problem
